libs/components/core/src/lib/modules/id/id.service.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { SkyIdService } from './id.service';
+
+describe('SkyIdService', () => {
+  function setupTest() {
+    const idService = new SkyIdService();
+    return { idService };
+  }
+
+  it('should generate a unique ID', () => {
```

Comment:
The ID directive already has tests that validate the logic for generating unique IDs. I'd move that test code into the new ID service unit tests, then update the ID directive's unit tests to take a mock ID service and just validate that it gets its ID from the ID service instead of duplicating the unique ID validation logic there.